### PR TITLE
DL-13464 Update to pekko log stream

### PR DESCRIPTION
**Pull request by jmountain-hmrc**

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
     <logger name="com.codahale.metrics.graphite.GraphiteReporter" level="OFF" />
 
-    <logger name="akka.stream" level="INFO" />
```
Comment (nadinJ) [inline conf/logback.xml:60]: PR Bot comments not addressed

File: conf/logback-test.xml [hunk 2]
```diff
@@ -34,14 +22,6 @@
...
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="com.google.inject" level="OFF"/>
 
     <logger name="org.asynchttpclient.netty" level="OFF"/>
```
Comment (paulanderson1234) [inline conf/logback-test.xml:27]: These entries relating to netty can also be removed:
logger name="org.asynchttpclient.netty" level="OFF" 
logger name="io.netty.buffer" level="OFF"/
logger name="play.core.netty" level="OFF"

File: conf/logback.xml [hunk 3]
```diff
@@ -14,35 +14,13 @@
...
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="com.google.inject" level="INFO"/>
 
     <logger name="org.asynchttpclient.netty" level="INFO"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:26]: This entry relating to netty can also be removed:
logger name="org.asynchttpclient.netty" level="INFO"

File: conf/logback-test.xml (hunk 2 above)
Comment (paulanderson1234) [reply conf/logback-test.xml]: All - removed - resolved

File: conf/logback.xml (hunk 3 above)
Comment (paulanderson1234) [reply conf/logback.xml]: Removed - resolved

**Review by paulanderson1234 — APPROVED** [review]

**Pull request merged**